### ISSUE TITLE
string column metrics bug

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#338]
I am sorry there was a bug in my codes for string column description and it adds NaN to the results. so to solve it before computing any number please remove the NaN data from the calculations. this command

```
import numpy as np
df['a'] = df['a'].replace(np.nan, '', regex=True)
```

see http://alphatechadmin.pythonanywhere.com/dtale/popup/describe/1?selectedCol=City most of the numbers have 74 (NaN or missing number)

**Comment by aschonfeld** [comment]
@reza1615 I've added `.fillna('')` to the code, but I'm not seeing the 74 for `selectedCol`.  Did you alter the data in some way to get that?

**Comment by aschonfeld** [comment]
@reza1615  I see 74 for the `Missing:` value but that seems correct.  I can validate that when applying the custom filter `City != City`.  Are you saying that these 74 nans should be updated to `''` before we start building your column metrics?  That seems a little off since these records are `nan` and not actually an empty string, but if thats how you think it should be treated thats fine with me.

**Comment by reza1615** [comment]
please check 
start with space
end with space
have punctuation 
theses cases the number 74 should be 0

**Comment by reza1615** [comment]
![Screenshot_20201212-130651_Samsung Internet](https://user-images.githubusercontent.com/2202483/101991490-ff30d880-3c7a-11eb-8262-b878bacee24c.jpg)

**Comment by reza1615** [comment]
yes it is right. we can remove 0 from the min result and get next min

**Comment by aschonfeld** [comment]
fixed in v1.28.0
